docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md

```diff
@@ -0,0 +1,361 @@
+# Dasharo Compatibility: Dasharo Tools Suite
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Wired network connection.
+1. Disable Secure Boot.
+1. Prepare the [bootable USB stick](../../../common-coreboot-docs/dasharo_tools_suite/#bootable-usb-stick)
+1. Make yourself familiar with
+    [Dasharo Tools Suite](../../../common-coreboot-docs/dasharo_tools_suite).
+
+## DTS001.001 Booting DTS from USB works correctly
+
+**Test description**
+
+This test aims to verify that DTS is properly booting from USB.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Plug the USB stick with DTS into the USB slot on the DUT.
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+
+**Expected result**
+
+After a while, the DTS menu should appear.
+
+## DTS002.001 DTS option Creating Dasharo HCL report works correctly
+
+**Test description**
+
+This test aims to verify that the option `Dasharo HCL report` in the DTS menu
+properly creates the report.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Plug the USB stick with DTS into the USB slot on the DUT.
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type in `1` and press `Enter`.
+1. Wait for the question: Do you want to support Dasharo development by sending
+    us logs with hardware configuration?
+1. Type in `y` and press Enter.
+
+**Expected result**
+
+The report should be generated, the whole process may take a few minutes.
```

Comment:
At this point, we should add information, that the test case ends with success if in the summary we could read that all calls exited without errors and the report has been sent to the cloud.